docs/concepts/testnets.md

```diff
@@ -8,7 +8,7 @@ This page documents all gno.land testnets, what their properties are, and how
 they are meant to be used. For testnet configuration, visit the 
 [reference section](../reference/network-config.md).
 
-Gno.land testnets are categorized by 4 main points:
```

Comment:
Isn't the brand guideline that `gno.land` at the start of the sentence is capitalized as `Gno.land`?

Reply:
We discussed internally and the conclusion was that we will write `gno.land` no matter what.